libknet/Makefile.am

```diff
@@ -113,7 +113,12 @@ endif
 if BUILD_COMPRESS_LZO2
 pkglib_LTLIBRARIES	+= compress_lzo2.la
 compress_lzo2_la_LDFLAGS = -module -avoid-version
+if KNET_LINUX
 compress_lzo2_la_CFLAGS	= $(lzo2_CFLAGS)
+endif
+if KNET_BSD
+compress_lzo2_la_CFLAGS = $(lzo2_CFLAGS) -I/usr/local/include
```

Comment:
Is `/usr/local/include` not on the default clang include path?

Reply:
Nope and only lzo2 is affected.

Reply:
Weird...
Wouldn't it be better to just tack that `-I/usr/local/include` to `compress_lzo2_la_CFLAGS` unconditionally, with a comment that clang on BSD needs this? You could leave out all the conditional bloat.
No objections either way.

Reply:
The issue is a combination of clang not including -I/usr/local/include, pkg-config lzo2 and lzo2 header files. Since it´s so specific to lzo2, I didn´t feel comfy to override it for all. In fact, the real fix would be for lzo2 pkg-config to report -I/usr/local/include instead of -I/usr/local/include/lzo2 in the cflags.

Reply:
@fabbione: Indeed, lzo 2.10 added pkg-config support! I asked the author to fix up this problem.
BTW I didn't mean overriding anything, just adding `-I/usr/local/include` to `compress_lzo2_la_CFLAGS` exactly as you did, but unconditionally with an explanation comment instead. It isn't obvious at all why it's done the way it's done now.